docs/schema/outlookitemcustomizations-element.md

```diff
@@ -30,101 +31,26 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
 
-<table>
-<colgroup>
-<col width="20%" />
-<col width="80%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>Attribute</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p>**ItemTypeDisplayName**</p></td>
-<td align="left"><p>The display name of the Outlook item type.</p>
-<p>Required.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p>**MessageClass**</p></td>
-<td align="left"><p>Used to identify the form used to display the item. It follows the format: `IPM.<OutlookItemType>.<ContentTypeName>`.</p>
-<p>Example: IPM.Contact.Customer</p>
-<p>Required.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p>**Description**</p></td>
-<td align="left"><p>A description for the customizations.</p>
-<p>Optional.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-</tbody>
-</table>
-
-<br/>
+|Attribute|Description|
```

Comment:
The description filed go reordered.  Please use this instead:

`|Attribute|Description|
|---------|-----------|
|**ItemTypeDisplayName**|The display name of the Outlook item type.</br></br>Required.</br></br>Attribute type: **String**|
|**MessageClass**|Used to identify the form used to display the item. It follows the format: `IPM.<OutlookItemType>.<ContentTypeName>`.</br></br>Example: IPM.Contact.Customer</br></br>Required</br></br>Attribute type: **String**|
|**Description**|A description for the customizations.</br></br>Optional.</br></br>Attribute type: **String**|`